### ISSUE TITLE
Form entry view sorting & pagination

### ISSUE THREAD

**Issue by k3rn3l3rr0r** [ufik/form-module#4]
There is a small glitch in the form entry view in the backend.

If there is more than $grid->setDefaultPerPage() entries in the grid the newest entries are shown on the second page and not as first entries on the first page... I've tried to play with different sorting settings and adjust the Grido or DataSource settings, but the changes are always affecting only the sort order inside of one page of the paginator. I was not able to switch the order to DESC for the whole list...

Maybe it has something to do with this?
*[2015-03-23 20-25-19] PHP Notice: Column with name 'date' is not sortable. in /var/www/[...]/libs/o5/grido/Grido/Grid.php:818*

**Comment by ufik** [comment]
This line says, that data grid should order items by date DESC.

https://github.com/ufik/form-module/blob/master/Admin/presenters/FormPresenter.php#L38

which is correct, isn't it?

**Comment by k3rn3l3rr0r** [comment]
It does partially work, the entries are sorted in DESC order, but only inside of the specific page of the paginator, not globally in the list.  If there are two pages, the newest entries are showing up on the second page (sorted DESC). I would await, the newest entries always to show up at the top of the first page, because of this line https://github.com/ufik/form-module/blob/master/Admin/presenters/FormPresenter.php#L38

**Comment by ufik** [comment]
I would try to add `setSortable` function like this ->setSortable()->setFilterDate().

**Comment by k3rn3l3rr0r** [comment]
Yes, that was my first approach... The default order is still the same, it
just makes the table header "clicable". If you try to click and change the
order it redirects but the order remains unchanged.
